policyengine_us/variables/gov/states/al/tax/income/deductions/al_federal_income_tax_deduction.py

```diff
@@ -10,5 +10,7 @@ class al_federal_income_tax_deduction(Variable):
     reference = "https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2023/01/22f40abk.pdf#page=20"
     defined_for = StateCode.AL
 
-    adds = "gov.states.al.tax.income.deductions.federal_tax.countable_sources"
-    subtracts = "gov.states.al.tax.income.deductions.federal_tax.credits"
+    adds = [
```

Comment:
we do not need the squared brackets here, only for variable inputs, same for the subtracts